.storybook/test-runner.ts

```diff
@@ -0,0 +1,31 @@
+import { type TestRunnerConfig, getStoryContext } from '@storybook/test-runner';
```

Comment:
Most files under `.storybook` folder has copyright, so copyright is needed?

Reply:
Thanks, you're right. I've added it now.